gnosis/cowswap
# blur bg on err pages

# Summary

Fixes #1485 

reproduce an err page an check the background

  # To Test

1. Open the page[ "Cow Game" ](https://pr1829--gpswapui.review.gnosisdev.com/#/play/cow-runner)to throw an error so we can see the app crashing.



![image](https://user-images.githubusercontent.com/622217/141478074-a8a6d518-4436-4a93-a1a1-464f788ead3e.png)
